base/compiler/ssair/domtree.jl

```diff
@@ -587,11 +587,20 @@ Checks if `bb1` dominates `bb2`.
 (Other blocks may be in between, e.g `bb1->bbx->bb2`).
 """
 dominates(domtree::DomTree, bb1::BBNumber, bb2::BBNumber) =
-    _dominates(domtree, bb1, bb2)
-postdominates(domtree::PostDomTree, bb1::BBNumber, bb2::BBNumber) =
-    _dominates(domtree, bb1, bb2)
+    generic_dominates(domtree, bb1, bb2)
 
-function _dominates(domtree::GenericDomTree, bb1::BBNumber, bb2::BBNumber)
+"""
+    post_dominates(domtree::DomTree, bb1::Int, bb2::Int) -> Bool
```

Comment:
`postdominate` is one word. `post_dominates` sounds like there's a `post` that dominates something

Reply:
Okay, sorry for the hasty change. I renamed it to the previous one.